### DISCUSSION TITLE
Migrating projects to cdt-cloud?

### DISCUSSION THREAD

**Discussion by MatthewKhouzam** [eclipse-cdt-cloud/cdt-cloud#8] [General]
Hi all,

I would like to know how to migrate certain projects to cdt-cloud? 

Project examples: 

- https://github.com/theia-ide/theia-trace-extension/
- https://github.com/theia-ide/timeline-chart/
- https://github.com/theia-ide/tsp-typescript-client

**Comment by jonahgraham** [comment]
I think you just raise a request on https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/new referencing https://projects.eclipse.org/projects/ecd.cdt.cloud which shows the "permission" for the move in place? That is what I plan to do for cdt-gdb-adapter and friends (as soon as I have a chance!)

**Reply by planger** [reply]
Exactly! You as a project lead can request the creation of one or more repositories for the new components in the [Github organization of CDT.Cloud](https://github.com/eclipse-cdt-cloud) via the [help desk]( https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/new). Note that they prefer batch requests for multiple repos rather than one ticket per repository to be created. If the new content is an initial or substantial contribution to Eclipse, a [contribution questionnaire (CQ)](https://projects.eclipse.org/projects/ecd.cdt.cloud/cq/create) might be needed in addition before pushing the code -- the friendly folks at the Eclipse Fnd helpdesk should be able to consult on whether a CQ is needed for a specific situation.

**Reply by jonahgraham** [reply]
I put in the request for CDT's project's now: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/793
